### PR TITLE
Implement AppleWatch into application

### PR DESCRIPTION
**Pull request by Tiewhan**
<h1>Overview</h1>
Add AppleWatch to iOS application. All it does is show the friend's profile picture when the friend's profile is pressed.

### CODE REVIEW

**Review by bgouws — APPROVED** [review]
All good.

File: Feisty/AppleWatchFiles/SessionHandler.swift [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  
+  func sessionDidDeactivate(_ session: WCSession) {
+    self.session.activate()
+  }
+  
+  func session(_ session: WCSession,
+               didReceiveMessage message: [String: Any],
+               replyHandler: @escaping ([String: Any]) -> Void) {
+    
+      if message["request"] as? String == "version" {
```
Comment (Zamo22) [inline Feisty/AppleWatchFiles/SessionHandler.swift]: What happens if this is not the case ?

File: FeistyWatch Extension/InterfaceController.swift [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  func session(_ session: WCSession, didReceiveMessage message: [String: Any]) {
+      
+    WKInterfaceDevice.current().play(.notification)
+    
+    setImage(message: message)
+    
+  }
+  
+  private func setImage(message: [String: Any]) {
+    guard let imageData = message["image"] as? NSData? else {
```
Comment (Zamo22) [inline FeistyWatch Extension/InterfaceController.swift]: Why do we convert this to NSData and then to Data ? Why not Data directly ?

File: Feisty/AppleWatchFiles/SessionHandler.swift (hunk 1 above)
Comment (Tiewhan) [reply Feisty/AppleWatchFiles/SessionHandler.swift]: this entire function shouldn't even exist, it's a remained from sample code. I'll remove it.

File: FeistyWatch Extension/InterfaceController.swift (hunk 2 above)
Comment (Tiewhan) [reply FeistyWatch Extension/InterfaceController.swift]: Hmm, it seems to be left over from the way I handled images with CoreData (that stored it as NSData). I'll fix it

**Review by Zamo22 — APPROVED** [review]

**Pull request merged**